trestle/tasks/csv_to_oscal_cd.py

```diff
@@ -54,6 +55,13 @@ def derive_part_id(control_mapping: str) -> str:
     return rval
 
 
+def non_empty(list_: List) -> int:
```

Comment:
Looking at the code it seems this function is used only to check whether the list is empty or not and if not then  append the list.
We have a function `as_list` that handles None and returns [] in such a case. Please use that instead

Reply:
Fixed.